app/components/product/quick-shop.tsx

```
@@ -1,174 +1,404 @@
+              } as React.CSSProperties
+            }
+            className={clsx([
+              "overflow-hidden",
+              "data-[state=closed]:animate-collapse",
+              "data-[state=open]:animate-expand",
+            ])}
+          >
+            <div
+              suppressHydrationWarning
+              className="prose dark:prose-invert py-2.5"
+              dangerouslySetInnerHTML={{ __html: content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)